.github/workflows/test.yml

```diff
@@ -0,0 +1,157 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - '*'
+      - '**'
+    tags-ignore:
+      - v*
+  pull_request:
+
+jobs:
+  test:
+    name: PHP
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: ['7.4']
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      #
+      # ENVIRONMENT DEPENDENCIES SETUP
+      #
+      - name: Configure PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: pcov
+
+      - name: Setup Yarn
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+
+      # Start MySQL and Create Databases
+      - name: Create Databases
+        run: |
+          sudo /etc/init.d/mysql start
+          mysql -e 'CREATE DATABASE IF NOT EXISTS nexus;' -h127.0.0.1 -uroot -proot
+
+      #
+      # COMPOSER DEPENDENICES
+      #
+
+      # Add Github Auth to Composer
+      - name: Add Composer GitHub Token
+        run: composer config -g github-oauth.github.com ${{ secrets.GITHUB_TOKEN }}
+
+      # Restore Caches
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: |
+          echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Restore Composer Cache
+        uses: actions/cache@v1
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+
+      - name: Restore Vendor From Cache
+        uses: actions/cache@v1
+        with:
+          path: vendor
+          key: ${{ runner.OS }}-build-${{ hashFiles('**/composer.lock') }}
+
+      # Install
+      - name: Install Composer Dependencies
+        run: composer install --prefer-dist --no-interaction --optimize-autoloader --no-suggest
+
+      #
+      # YARN DEPENDENCIES
+      #
+
+      # Restore Caches
+      - name: Get Yarn Cache Directory
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Restore Yarn Cache
+        uses: actions/cache@v1
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      # Install
+      - name: Install Core Assets
```

Comment:
Is this core as in important or core as in the name? If so, might need renaming

Reply:
Was a typo! Now corrected